tools/openshift/ocp-ipi-aws/prep_for_subm.sh

```diff
@@ -1,82 +1,80 @@
 #!/bin/sh
 set -e
+
+# Script args
 OCP_INS_DIR="$(realpath ${1:-.})"
+(( $# == 1 )) || shift && TERRAFORM_ARGS=$@ # e.g. -auto-approve -lock-timeout=3m
 METADATA_JSON="${OCP_INS_DIR}/metadata.json"
+
+# Github args
 GITHUB_BRANCH="${GITHUB_BRANCH:-master}"
 GITHUB_USERFORK="${GITHUB_USERFORK:-submariner-io}"
-GITHUB_ZIP="${GITHUB_ZIP:-https://github.com/$GITHUB_USERFORK/submariner/archive/$GITHUB_BRANCH.zip}"
+GITHUB_ARCHIVE="https://github.com/$GITHUB_USERFORK/submariner/archive/$GITHUB_BRANCH.tar.gz"
+
 
 req_install() {
-        if ! command -v $1 >/dev/null 2>&1; then
-	    echo "$1 is required by this tool, please install $1" >&2
-	    exit 1
-	fi
+  if ! command -v $1 >/dev/null 2>&1; then
+    echo "$1 is required by this tool, please install $1" >&2
+    exit 1
+  fi
 }
 
 download_ocp_ipi_aws_tool() {
-	TMP=$(mktemp -d)
-	IPI_AWS="submariner-${GITHUB_BRANCH}/tools/openshift/ocp-ipi-aws"
-	curl -L "${GITHUB_ZIP}" --output $TMP/submariner.zip
-	unzip $TMP/submariner.zip "$IPI_AWS/*" -d $TMP
-	cp -rfp $TMP/$IPI_AWS .
-	rm -rf $TMP
+  echo "Downloading $GITHUB_ARCHIVE and extracting 'ocp-ipi-aws' tool"
+  wget -O - $GITHUB_ARCHIVE | tar xz --strip=4 "submariner-${GITHUB_BRANCH}/tools/openshift/ocp-ipi-aws"
 }
 
 # check pre-requisites
 
-for cmd in unzip terraform oc aws; do
-	req_install $cmd
+for cmd in terraform oc aws; do
```

Comment:
This could check for `wget` too, since that’s not installed by default everywhere:

```suggestion
for cmd in wget terraform oc aws; do
```